app/models/miq_request.rb

```diff
@@ -35,12 +35,16 @@ class MiqRequest < ApplicationRecord
   virtual_column  :stamped_on,           :type => :datetime, :uses => :miq_approvals
   virtual_column  :v_allowed_tags,       :type => :string,   :uses => :workflow
   virtual_column  :v_workflow_class,     :type => :string,   :uses => :workflow
+  virtual_column  :v_supports_pxe?,      :type => :string,   :uses => :workflow
+  virtual_column  :v_supports_iso?,      :type => :string,   :uses => :workflow
   virtual_column  :request_type_display, :type => :string
   virtual_column  :resource_type,        :type => :string
   virtual_column  :state,                :type => :string
 
   delegate :allowed_tags,                :to => :workflow,   :prefix => :v,  :allow_nil => true
   delegate :class,                       :to => :workflow,   :prefix => :v_workflow
+  delegate :supports_pxe?,               :to => :workflow,   :prefix => :v
```

Comment:
@kbrock Thanks.

Not sure what I'm missing here, but this is what I'm trying:
```ruby
has_one :workflow
virtual_delegate :supports_pxe,        :to => "workflow.supports_pxe?",    :allow_nil => true
virtual_delegate :supports_iso,        :to => "workflow.supports_iso?",    :allow_nil => true
```

When I try the below API, I do not see an output for `supports_pxe`
```
api/requests/60?attributes=supports_pxe
```